src/components/Categories/CategoryList/CategoryList.module.css

```diff
@@ -0,0 +1,74 @@
+.categoriesList {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.categoryItem {
+  height: 250px;
+  border-radius: 20px;
+  position: relative;
+  background: rgba(5, 5, 5, 0.18);
+  background-image: url('../../../temp/cat-img.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.categoryLinkContainer {
+  position: absolute;
+  bottom: 16px;
+  left: 16px;
+  display: flex;
+  column-gap: 4px;
+  align-items: center;
+}
+
+.categoryLink {
+  height: 40px;
+  padding: 0 12px;
+  border: 1px solid var(--color-white-40);
+  border-radius: 30px;
+  background: var(--color-white-30);
+  display: flex;
+  align-items: center;
+  font-family: var(--font-family);
```

Comment:
should be add to css/reset.css. remove please

Reply:
corrected